src/commands/Staff/punishment.ts

```diff
@@ -665,7 +666,7 @@ const command: Command = {
 
 async function calculatePunishment(user: Discord.User, points: PunishmentPoints): Promise<Punishment> {
 	const activePunishments = await getActivePunishments(user),
-		hasActivePunishment = activePunishments.some(p => p.ended === false),
+		activePunishmentPoints = activePunishments.find(p => p.ended === false)?.points || null,
```

Comment:
Although this doesn’t actually change anything, nullish coalescing is better
```suggestion
		activePunishmentPoints = activePunishments.find(p => p.ended === false)?.points ?? null,
```